Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:6
+
+# Export the Websocket port for Flowhub connection
+EXPOSE 3569
+
+# Reduce npm install verbosity, overflows Travis CI log view
+ENV NPM_CONFIG_LOGLEVEL warn
+# Configure Bitraf MQTT
+ENV MSGFLO_BROKER mqtt://mqtt.bitraf.no
```

Comment:
This should not be default, this is the production system, only one instance can use that.